src/arr/compiler/ast-util.arr

```diff
@@ -1133,11 +1145,17 @@ fun get-named-provides(resolved :: CS.NameResolution, uri :: URI, compile-env ::
             exp = resolved.datatypes.get-value-now(d.d.key())
             data-typs.set-now(d.d.key(), data-expr-to-datatype(exp))
           end
+          module-typs = SD.make-mutable-string-dict()
+          for each(m from modules):
+            mod = resolved.modules.get-value-now(m.name.key())
+            module-typs.set-now(m.name.toname(), CS.m-resolved(mod))
```

Comment:
This should probably be `CS.m-uri(mod.from-uri)` to avoid copying exports everywhere.